pkgs/applications/networking/browsers/offpunk/default.nix

```diff
@@ -0,0 +1,46 @@
+{stdenv, testVersion, makeWrapper, python3, lib, fetchFromGitea, timg, less, offpunk, ...}:
+
+let
+  pythonDependencies = (with python3.pkgs; [
+    beautifulsoup4
+    cryptography
+    feedparser
+    pillow
+    readability-lxml
+    requests
+    setproctitle
+  ]);
+  otherDependencies = [ timg less ];
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "offpunk";
+  version = "1.4";
+
+  src = fetchFromGitea {
+    domain = "notabug.org";
+    owner = "ploum";
+    repo = "offpunk";
+    rev = "v${finalAttrs.version}";
+    sha256 = "04dzkzsan1cnrslsvfgn1whpwald8xy34wqzvq81hd2mvw9a2n69";
+  };
+
+  buildInputs = [ makeWrapper ] ++ otherDependencies ++ pythonDependencies;
+
+  installPhase = ''
+    install -D ./offpunk.py $out/bin/offpunk
```

Comment:
Use the `runHook` decorations.

Reply:
If you meant `runHook preInstall` and `runHook postInstall`, it is now done. If you meant something else, please tell me.

Reply:
Yes, I meant that..